api/tests/unit/segments/test_unit_segments_models.py

```diff
@@ -150,6 +150,48 @@ def test_delete_segment_only_schedules_one_task_for_audit_log_creation(
     assert len(mocked_tasks.mock_calls) == 1
 
 
+def test_system_segment_get_skip_create_audit_log(system_segment: Segment) -> None:
+    # When
+    result = system_segment.get_skip_create_audit_log()
+
+    # Then
+    assert result is True
+
+
+def test_system_segment_rule_get_skip_create_audit_log(system_segment: Segment) -> None:
+    # Given
+    segment_rule = SegmentRule.objects.create(
+        segment=system_segment,
+        type=SegmentRule.ALL_RULE,
+    )
+
+    # When
+    result = segment_rule.get_skip_create_audit_log()
+
+    # Then
+    assert result is True
+
+
+def test_system_segment_condition_get_skip_create_audit_log(
+    system_segment: Segment,
+) -> None:
+    # Given
+    segment_rule = SegmentRule.objects.create(
+        segment=system_segment,
+        type=SegmentRule.ALL_RULE,
+    )
+    condition = Condition.objects.create(
+        rule=segment_rule,
+        property="foo",
+        operator=EQUAL,
+        value="bar",
+    )
+    # When
+    result = condition.get_skip_create_audit_log()
+    # Then
+    assert result is True
```

Comment:
Those 2 tests should be redundant with [last week changes](https://github.com/Flagsmith/flagsmith/pull/5898/files#diff-c911f59cd1a3ea2aa55ef1db5cb21f5b777cf28612604ab7ae7754fea36eb0f8R32-R43)